Userland/Applications/Piano/PlayerWidget.cpp

```diff
@@ -16,9 +17,9 @@
 #include <LibGUI/ItemListModel.h>
 #include <LibGUI/Label.h>
 
-ErrorOr<NonnullRefPtr<PlayerWidget>> PlayerWidget::create(TrackManager& manager, AudioPlayerLoop& loop)
+ErrorOr<NonnullRefPtr<PlayerWidget>> PlayerWidget::try_create(TrackManager& manager, MainWidget& main_widget, AudioPlayerLoop& loop)
```

Comment:
Only use `try_create`, when there is an infallible version of it as well
see #17182

Reply:
This name is required by `Object::add`

Reply:
ah ok